doc/ha-squid-migration.md

```diff
@@ -0,0 +1,207 @@
+      egress.3505169447.cidr_blocks.0:      "192.168.144.0/20" => ""
+      egress.3505169447.cidr_blocks.1:      "54.224.0.0/12" => ""
+      egress.3505169447.description:        "" => ""
+      egress.3505169447.from_port:          "0" => "0"
+      egress.3505169447.ipv6_cidr_blocks.#: "0" => "0"
+      egress.3505169447.prefix_list_ids.#:  "0" => "0"
+      egress.3505169447.protocol:           "-1" => ""
+      egress.3505169447.security_groups.#:  "0" => "0"
+      egress.3505169447.self:               "false" => "false"
+      egress.3505169447.to_port:            "0" => "0"
+      tags.%:                               "2" => "3"
+      tags.Name:                            "" => "generic-commons-local-sec-group"
+
+  ~ module.cdis_vpc.aws_security_group.out
+      tags.%:                               "2" => "3"
+      tags.Name:                            "" => "generic-commons-outbound-traffic"
+
+  - module.cdis_vpc.aws_security_group.proxy
+
+  - module.cdis_vpc.aws_security_group.webservice
+
+  - module.cdis_vpc.module.squid_proxy.aws_ami_copy.squid_ami
+
+  - module.cdis_vpc.module.squid_proxy.aws_eip.squid
+
+  - module.cdis_vpc.module.squid_proxy.aws_eip_association.squid_eip
+
+  - module.cdis_vpc.module.squid_proxy.aws_instance.proxy
+
+  - module.cdis_vpc.module.squid_proxy.aws_security_group.login-ssh
+
+  - module.cdis_vpc.module.squid_proxy.aws_security_group.out
+
+  - module.cdis_vpc.module.squid_proxy.aws_security_group.proxy
+
+
+Plan: 3 to add, 3 to change, 17 to destroy.
+```
+
+
+If it doesn't look like above, make sure the no harm will be done to production clusters. Then apply the plan.
+
+
+```bash
+gen3 tfapply
+```
+
+
+### 2. Update the EKS module
+
+
+Start by initializing the module:
+
+```bash
+gen3 workon cdistest generic_commons_eks
+gen3 cd
+```
+
+
+We need to also taint a resource for the eks module, for that, run the following:
+
+```bash
+gen3 tform taint -module eks aws_route_table.eks_private
+```
+
+When that's done, you may proceed as usual
+
+```bash
+gen3 tfplan
+```
+
+The plan for the EKS module should not delete any resource other than the one you just tainted. If otherwise, please check the plan thoroughly.
+
+
+```bash
+Plan: 22 to add, 8 to change, 1 to destroy.
+```
+
+Apply the plan
+
+```bash
+gen3 tfapply
+```
+
+
+
+## Considerations
+
+During the migration, when you first update the VPC module, the proxy will go away, leaving you temporarily without a connection to the internet from the worker nodes, you may want to plan accordingly. Nonetheless, services will be available to users, but if you use an internal authentication provider, like google, dbGap, among others, these ones may not work until there is an available proxy.
+
+After you update the EKS module the Squid instances may take a few minutes to fully start serving proxy services, Say it might take up to 20 minutes.
+
+There might not be any roll back procedure after you are done.
```

Comment:
This is great!
Would you add a link to the `doc/README.md` file too - ex:
* [ha-squid](./hq-squid.md)
Then `gen3 help squid` will give that as a search result ...